run_principal.py

```diff
@@ -19,37 +19,51 @@
 import sys
 import getopt
 
+workspace = ''
+conditionsPath = r'./condiciones_iniciales/'
+parametersFile = 'parameters.xlsx'
+result_name = 'model_time_series.csv'
+
+if sys.argv[1:]:
+    opts, args = getopt.getopt(sys.argv[1:], "o:w:")
+    outputName = opts[0][1]
+    workspace = opts[1][1]
+    parametersFile = f'{workspace}_parameters.xlsx'
+    result_name = f'{workspace}_{outputName}'
```

Comment:
I tested it from the simulator container and it expects -o always first, I think it would be better to assign values with the for as it was before (just without the brake)